src/tests/unit/containers/orders/cart/path-back/path-back.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import PathBack from '../../../../../../containers/orders/cart/path-back/path-back';
+
+describe('Path-back component tests', () => {
+  const history = createMemoryHistory();
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = render(
+      <Router history={history}>
+        <PathBack />
+      </Router>
+    );
+  });
+
+  it('should render PathBack', () => {
+    expect(wrapper).toBeDefined();
```

Comment:
Can we check if some function will be working out when we click on this component?

Reply:
It' s a simple component that only includes links.So, I check the render component and path by directive getByText.